modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/intent_to_file_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require_relative '../../../rails_helper'
+
+RSpec.describe AccreditedRepresentativePortal::V0::IntentToFileController, type: :request do
+  let(:poa_code) { '067' }
+  let(:poa_check_vcr_response) { '200_response' }
+  let!(:test_user) do
+    create(:representative_user, email: 'test@va.gov', icn: '123498767V234859')
+  end
+  let!(:accredited_individual) do
+    create(:user_account_accredited_individual,
+           user_account_email: test_user.email,
+           user_account_icn: test_user.icn,
+           accredited_individual_registration_number: '357458',
+           poa_code:)
+  end
+  let!(:vso) { create(:organization, poa: poa_code, can_accept_digital_poa_requests: true) }
+
+  let!(:representative) do
+    create(:representative,
+           :vso,
+           representative_id: accredited_individual.accredited_individual_registration_number,
+           poa_codes: [poa_code])
+  end
+
+  before do
+    allow_any_instance_of(Auth::ClientCredentials::Service).to receive(:get_token).and_return('fake_access_token')
+    login_as(test_user)
+  end
+
+  around do |example|
+    VCR.use_cassette("lighthouse/benefits_claims/power_of_attorney/#{poa_check_vcr_response}") do
+      example.run
+    end
+  end
+
+  describe 'GET /accredited_representative_portal/v0/intent_to_file' do
+    context 'feature flag is off' do
+      it 'returns forbidden' do
+        Flipper.disable(:accredited_representative_portal_intent_to_file_api)
```

Comment:
Can you make sure you have tests for the flipper on and flipper off scenarios? 

Can you also use mocking instead of disabling directly in the test? disabling or enabling can cause other tests to fail when specs are run in parallel. See details in [this backend example](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample).